src/django_components/templatetags/component_tags.py

```diff
@@ -278,23 +242,11 @@ def do_html_attrs(parser: Parser, token: Token) -> HtmlAttrsNode:
     ```
     """
     bits = token.split_contents()
-    attributes, default_attrs, append_attrs = parse_html_attrs_args(parser, bits, "html_attrs")
+    attributes, default_attrs, append_attrs = _parse_html_attrs_args(parser, bits, "html_attrs")
     return HtmlAttrsNode(attributes, default_attrs, append_attrs)
 
 
-def is_whitespace_node(node: Node) -> bool:
```

Comment:
These three `is_whitespace_node`,  `is_whitespace_token` and `is_block_tag_token` weren't being used. Actually, even in the MR that introduced them didn't use them. Since it's not documented, I assume it's not part of the public API.